src/clientManager.js

```diff
@@ -110,7 +110,7 @@ ClientManager.prototype.removeUser = function(name) {
 ClientManager.prototype.autoload = function(/* sockets */) {
 	var self = this;
 	setInterval(function() {
-		var loaded = _.pluck(
+		var loaded = _.map(
 			self.clients,
```

Comment:
Not a bad idea, however I would get the upgrade out of the way, and then look into using vanillajs.